frontend/pages/branch/[id].tsx

```diff
@@ -14,30 +14,29 @@ import MembersTable from '../../src/modules/branch/components/MembersTable';
 import TeamsTable from '../../src/modules/branch/components/TeamsTable';
```

Comment:
Add relative paths for consistency with other imports.

<img width="476" alt="image" src="https://github.com/harunurkst/qard-al-hasana/assets/36256353/40f41c5e-ebc4-4797-bd75-f73ff20b4c17">